CHANGELOG.rst

```diff
@@ -1,6 +1,55 @@
 Changelog
 =========
 
+0.6.0 (2018-09-10)
+------------------
+
+MLflow 0.6.0 introduces two major features: a Java MLflow Client API published to Maven and inherent support for saving and serving SparkML models as MLeap for low-latency serving. In addition to these features, there are a host of improvements and bugfixes to the REST API, Python API, tracking UI, and documentation.
+
+Breaking changes:
+
+We fixed a few inconsistencies in the the ``mlflow.tracking.MlflowService`` introduced in 0.5.0:
+
+- ``MlflowService.list_runs`` was changed to ``MlflowService.list_run_infos`` to reflect the information actually returned by the call. It now returns a ``RunInfo`` instead of a ``Run`` (#334, @aarondav)
+- ``MlflowService.log_artifact`` and ``MlflowService.log_artifacts`` now take a ``run_id`` instead of ``artifact_uri``. This now matches ``list_artifacts`` and ``download_artifacts``  (#444, @aarondav)
+
+Features:
+
+- Java MLflow Client API added with support for the MLflow Tracking API (analogous to ``mlflow.tracking``), allowing users to create and manage experiments, runs, and artifacts. This includes a `usage example <https://github.com/mlflow/mlflow/blob/master/mlflow/java/client/src/main/java/org/mlflow/tracking/samples/QuickStartDriver.java>`_ and `Javadocs <https://mlflow.org/docs/latest/java_api/index.html>`_. The client is published to Maven under `mlflow:mlflow <https://mvnrepository.com/artifact/mlflow/mlflow>`_ (#380, #394, #398, #409, #410, #430, #452, @aarondav)
```

Comment:
Javadoc URI will work once we publish the docs. Maven repo deployment still in progress.